src/serialize/index.test.ts

```diff
@@ -59,6 +59,7 @@ describe('printElement', () => {
       `"<div>first second third </div>"`,
     );
     outerEl.style.whiteSpace = 'pre';
+    // eslint-disable-next-line @cloudfour/unicorn/template-indent
```

Comment:
I had to disable this rule in this file because the output after running the lint did not pass the tests, and the output after running the tests did not pass the lint. The eslint rule doesn't handle blank lines in a way that is compatible with Jest's snapshot checker.